client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -25,14 +25,14 @@ class SimplePaymentsView extends Component {
 			return ( <QuerySimplePayments siteId={ siteId } productId={ productId } /> );
 		}
 
-		const { title, description, price, currency } = product;
+		const { title, description, price, currency, multiple } = product;
+
+		const paypalButtonImageUrl = '/calypso/images/simple-payments/preview-paypal-button-gray.png';
```

Comment:
Good catch. I'm not 100% sure, but I doubt we have a vector version of the button as it's from PayPal. If that's correct, we should use double size image.

For small screens, let's move the button underneath of the qty field.